### PR TITLE
Remove Path directory in .yml file

### PR DESCRIPTION
**Pull request by sherrif10**
Removed path to   "qaframework-bdd-tests/src/**/*.*"
                            - "qaframework-bdd-tests/package.json"
                            
Since we only deal with 2.x specifically.

### CODE REVIEW

File: .github/workflows/refapp-2x-clinical-visit.yml [hunk 1]
```diff
@@ -4,9 +4,7 @@ on:
     branches: [master]
   pull_request:
     branches: [master]
-    paths:
-      - "qaframework-bdd-tests/src/**/*.*"
-      - "qaframework-bdd-tests/package.json"
+
```
Comment (kdaud) [inline .github/workflows/refapp-2x-clinical-visit.yml:7]: I think we can get rid of this line here

**Review by kdaud — COMMENTED** [review]
Thanks @sherrif10 . I have just left one minor comment to catch up and have this commit in.

File: .github/workflows/refapp-2x-clinical-visit.yml (hunk 1 above)
Comment (sherrif10) [reply .github/workflows/refapp-2x-clinical-visit.yml:7]: Resolved

Comment (kdaud) [reply .github/workflows/refapp-2x-clinical-visit.yml:7]: > Resolved

Is there any commit made to resolve the review above?

Comment (sherrif10) [reply .github/workflows/refapp-2x-clinical-visit.yml]: Opps sorry my internet had not pushed this.

**Review by kdaud — APPROVED** [review]

**Pull request merged**